port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -154,6 +154,10 @@ async def _unregister_resource_raw(
         results: list[RAW_ITEM],
         user_agent_type: UserAgentType,
     ) -> tuple[list[Entity], list[Exception]]:
+        if not isinstance(resource.port.entity.mappings.identifier, str):
+            logger.info("Skip unregistering due to non string identifier mapping")
```

Comment:
missing additional context for who this will be skipped ( which resource kind ).
and what would be the effect?